src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
I guess it's not a big deal, but doesn't this already make OpenSeadragon a global since it's outside of the closure? Another option would be to move this inside the closure, and also the UMD bit at the bottom.